emcc.py

```diff
@@ -1359,7 +1361,7 @@ def compile_source_file(i, input_file):
         assert len(temp_files) == len(input_files)
 
         # Optimize source files
-        if options.llvm_opts > 0:
+        if '-O0' not in options.llvm_opts:
```

Comment:
`'-O0' not in options.llvm_opts` appears twice, and I think there's a chance we may want to change it in the future. How about using a helper function that returns an integer optimization level, maybe called `get_llvm_opt_level()`?

Reply:
`llvm_opts` can also receive `-Os` and `-Oz` so it can't always be integer. Maybe just return `-O0`, `-O1`, etc?

Reply:
yeah, good point. how about "optimizing", so it returns a boolean of false just for `-O0`, true for everything else?

Reply:
what did we decide here? I still think a helper function like `is_optimizing()` makes sense. i don't have a good idea for the name though, so maybe not.

Reply:
Oops, done! Sorry for the late response.